appium-tests/common/common.spec.js

```diff
@@ -114,14 +116,15 @@ describe('Contacts Android', function () {
             })
             .click()
             .context(webviewContext)
-            .executeAsync(function (pID, cb) {
+            .executeAsync(function (pID, getOnlyBirthday, cb) {
```

Comment:
Why do you need to pass the `getOnlyBirthday` flag? Is there an issue with just always returning the full contact and checking the birthday field as part of the birthday spec?